diff --git a/content/renderer/media/media_interface_factory.cc b/content/renderer/media/media_interface_factory.cc
index f457c3854d8a95c29c3b1de5b811cb72b7ad6c6c..f8717420ca64183194278cdaa22a70eaa179cdaa 100644
--- a/content/renderer/media/media_interface_factory.cc
+++ b/content/renderer/media/media_interface_factory.cc
@@ -71,6 +71,7 @@ void MediaInterfaceFactory::CreateRenderer(
 void MediaInterfaceFactory::CreateCdm(
     const std::string& key_system,
     media::mojom::ContentDecryptionModuleRequest request) {
+  LOG(ERROR) << "BSC]] MediaInterfaceFactory::CreateCdm";
   if (!task_runner_->BelongsToCurrentThread()) {
     task_runner_->PostTask(
         FROM_HERE, base::BindOnce(&MediaInterfaceFactory::CreateCdm, weak_this_,
@@ -104,7 +105,7 @@ void MediaInterfaceFactory::CreateCdmProxy(
 
 media::mojom::InterfaceFactory*
 MediaInterfaceFactory::GetMediaInterfaceFactory() {
-  DVLOG(1) << __func__;
+  LOG(ERROR) << "BSC]] " << __func__;
   DCHECK(task_runner_->BelongsToCurrentThread());
 
   if (!media_interface_factory_) {
@@ -117,7 +118,7 @@ MediaInterfaceFactory::GetMediaInterfaceFactory() {
 }
 
 void MediaInterfaceFactory::OnConnectionError() {
-  DVLOG(1) << __func__;
+  LOG(ERROR) << "BSC]] " << __func__;
   DCHECK(task_runner_->BelongsToCurrentThread());
 
   media_interface_factory_.reset();
