diff --git a/content/browser/media/cdm_registry_impl.cc b/content/browser/media/cdm_registry_impl.cc
index 81936b7601aa72effc5f99a188da72f8b9469568..203cc67c7c999e3d314815b58e7ca631eb94b113 100644
--- a/content/browser/media/cdm_registry_impl.cc
+++ b/content/browser/media/cdm_registry_impl.cc
@@ -27,6 +27,7 @@ CdmRegistryImpl::CdmRegistryImpl() {}
 CdmRegistryImpl::~CdmRegistryImpl() {}
 
 void CdmRegistryImpl::Init() {
+  LOG(ERROR) << "BSC]] CdmRegistryImpl::Init";
   // Let embedders register CDMs.
   GetContentClient()->AddContentDecryptionModules(&cdms_, nullptr);
 }
@@ -37,6 +38,9 @@ void CdmRegistryImpl::RegisterCdm(const CdmInfo& info) {
   // use the same registered CDM for a given key system. This also means that
   // some later registered CDMs (component updated) will not be used until
   // browser restart, which is fine in most cases.
+  LOG(ERROR) << "BSC]] CdmRegistryImpl::RegisterCdm: "
+    << "name=" << info.name
+    << "; path=" << info.path;
   cdms_.push_back(info);
 }
 
