diff --git a/media/media_options.gni b/media/media_options.gni
index d8e20f62c19d34d036760883a02d3350aa696f1b..1dbaf1cfdc1d45c429ecb1e1485e5ccc29ba00c0 100644
--- a/media/media_options.gni
+++ b/media/media_options.gni
@@ -6,6 +6,7 @@ import("//build/config/chrome_build.gni")
 import("//build/config/chromecast_build.gni")
 import("//build/config/features.gni")
 import("//testing/libfuzzer/fuzzer_test.gni")
+import("//brave/build/config.gni")
 
 # Do not expand this list without double-checking with OWNERS, this is a list of
 # all targets which roll up into the //media component. It controls visibility
@@ -130,6 +131,7 @@ declare_args() {
   # remote process (e.g. "utility" process, see |mojo_media_host|).
   enable_mojo_media =
       is_android || is_chromecast || is_mac || enable_library_cdms || is_win
+  print("BSC]] enable_mojo_media=$enable_mojo_media")
 
   # Enable the TestMojoMediaClient to be used in mojo MediaService. This is for
   # testing only and will override the default platform MojoMediaClient, if any.
@@ -146,7 +148,7 @@ declare_args() {
   # Enables host verification for CDMs.
   # Windows and Mac.
   enable_cdm_host_verification =
-      enable_library_cdms && (is_mac || is_win) && is_chrome_branded
+      enable_library_cdms && (is_mac || is_win) && brave_enable_cdm_host_verification
 
   # Enable Storage ID which is used by CDMs. This is only available with chrome
   # branding, but may be overridden by other embedders.
@@ -197,6 +199,7 @@ if (enable_mojo_media) {
   }
 
   if (enable_library_cdms) {
+    print("BSC]] adding `cdm` to _default_mojo_media_services")
     _default_mojo_media_services += [ "cdm" ]
 
     # Having a CDM running means it could require a CdmProxy running in the GPU
