diff --git a/media/cdm/cdm_adapter_factory.cc b/media/cdm/cdm_adapter_factory.cc
index 8826fa5ec12e1a4c922287bc5c10bff7c421cc9d..7b547c980afe48211b3dfd1fea660006faf9016d 100644
--- a/media/cdm/cdm_adapter_factory.cc
+++ b/media/cdm/cdm_adapter_factory.cc
@@ -29,7 +29,7 @@ void CdmAdapterFactory::Create(
     const SessionKeysChangeCB& session_keys_change_cb,
     const SessionExpirationUpdateCB& session_expiration_update_cb,
     const CdmCreatedCB& cdm_created_cb) {
-  DVLOG(1) << __func__ << ": key_system=" << key_system;
+  LOG(ERROR) << __func__ << ": key_system=" << key_system;
 
   if (security_origin.unique()) {
     LOG(ERROR) << "Invalid Origin: " << security_origin;
